docker/build_base/Cargo.toml

```diff
@@ -17,9 +17,9 @@ arrow-schema = { version = "43.0.0" }
 parquet = { version = "43.0.0" }
 
 [patch.crates-io]
-parquet = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '43.0.0/arroyo_patches'}
-arrow = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '43.0.0/arroyo_patches'}
-arrow-buffer = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '43.0.0/arroyo_patches'}
-arrow-array = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '43.0.0/arroyo_patches'}
-arrow-schema = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '43.0.0/arroyo_patches'}
-object_store = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '43.0.0/arroyo_patches'}
+parquet = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '43.0.0/arroyo_patches-1'}
```

Comment:
is this something that should be merged in before committing?

Reply:
I cut to a different branch to avoid caching issues. Is there a better approach?

Reply:
You mean lockfile issues?

I don't think that will affect users, since they have our lockfiles. We may want to start checking in the build_dir lockfile for us.